### ISSUE TITLE
Editor Idea: Better Parameters Values

### ISSUE THREAD

**Issue by Blazulite** [ddnet/ddnet#7407]
At the moment, you can only put values like 100, 83, 91... It would be really cool if you were able to put values like 94.1, 191.18

(Only for tiles tho, not for entities for obvious reasons)

**Comment by Robyt3** [comment]
I don't think this is feasible with our current map format. Can you explain the motivation for this proposed change?

**Comment by Blazulite** [comment]
would be useful for a better parallax control

**Comment by Robyt3** [comment]
This is very unlikely to be realised with the current map format.
